### PR TITLE
Update scaffolding

### PR DESCRIPTION
**Pull request by dariobro**
## Describe your Pull Request
A clear and concise description of what the Pull Request is about.

Fixes # (issue)

Please select relevant options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

Reviewer: @nmeisenzahl

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -78,6 +78,20 @@ terraform init -input=false \
   -backend-config="storage_account_name=$saName" \
   -backend-config="container_name=$scName"
 ```
+## Azuread provider configuration
+```
+terraform {
+  required_providers {
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "=1.5.0"
```
Comment (nmeisenzahl) [inline Readme.md:87]: version = ">=1.5.0" - not production ready but helps users to get started with the newest version

File: up.sh [hunk 2]
```diff
@@ -104,7 +85,8 @@ az deployment group create \
                  "sc_name=$scName" \
                  "tenant_id=$tenantId" \
                  "user_id=$userId" \
-                 "tag"=$tag
+                 "tag"=$tag \
+                 "location"=$arm_location
```
Comment (nmeisenzahl) [inline up.sh:89]: Can we reuse the existing location variable?

**Comment by nmeisenzahl** [comment]
Can we please also update:
1. azurerm version in [Readme.md](https://github.com/whiteducksoftware/terraform-scaffold-for-azure/blob/master/Readme.md#scaffold-a-terraform-project)
2. We should make the Stac SKU variable uns use ZRS as the default
3. Did we find a quick fix for the local user query issue when using a guest account? If so, we could also fix this.

Can you please also deploy everything and test if everything works as expected?

**Comment by dariobro** [comment]
Update API permissions -> add permission to manage Applications

**Pull request merged**